<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1041</width>
    <height>805</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Parallax</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../img/sextant.png</normaloff>../img/sextant.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: rgb(00,00,00);
color: #FFFFFF;
}
QPushButton{
	background-color: black;
}
 QPushButton:pressed {
     background-color: rgb(224, 0, 0);     
}
QPushButton:hover {
     background-color: rgb(100, 30, 30);
}
QPushButton#startButton:disabled:checked {
	color: gray;
}
QPushButton#startButton:disabled:checked {
	background-color: #ffaaaa;
}
QPushButton#startButton:disabled:!checked {
	background-color: lightGreen;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="startButton">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Terminal</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">/* Disabled &amp; Checked */
QPushButton#startButton:checked {
	color: gray;
	background-color: #ffaaaa;
}

</string>
          </property>
          <property name="text">
           <string>START</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="recordButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">/* Disabled &amp; Checked */
QPushButton#recordButton:checked {
	color: gray;
	background-color: #ffaaaa;
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../img/recordingButton.png</normaloff>../img/recordingButton.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="snapshotButton">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">/* Disabled &amp; Checked */
QPushButton#snapshotButton:pressed {
	color: gray;
	background-color: #ffaaaa;
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../img/snapshotButton_white.png</normaloff>../img/snapshotButton_white.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>18</width>
            <height>18</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="dirLabel">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>C:\</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="browseDirButton">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QToolButton#browseDirButton:pressed {
	color: gray;
	background-color: #ffaaaa;
}

</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>68</width>
            <height>22</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="columnsLabel">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Terminal</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Columns:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="nColumnsSpinBox">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>40</height>
           </size>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>6</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1021</width>
         <height>694</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1041</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
